pkg/resource/aws/aws_instance_test.go

```diff
@@ -14,6 +14,35 @@ import (
 	"github.com/r3labs/diff/v2"
 )
 
+func TestAcc_AwsInstance(t *testing.T) {
+	acceptance.Run(t, acceptance.AccTestCase{
+		TerraformVersion: "0.14.9",
+		Paths:            []string{"./testdata/acc/aws_instance_default"},
+		Args: []string{
+			"scan",
+			"--filter",
+			"Type=='aws_instance' || Type=='aws_ebs_volume'",
+			"--tf-provider-version",
+			"3.45.0",
+		},
+		Checks: []acceptance.AccCheck{
+			{
+				Env: map[string]string{
+					"AWS_REGION": "us-east-1",
+				},
+				Check: func(result *test.ScanResult, stdout string, err error) {
+					if err != nil {
+						t.Fatal(err)
+					}
+					result.AssertDeletedCount(0)
+					result.AssertManagedCount(2)
+					result.AssertDriftCountTotal(0)
```

Comment:
```
result.AssertInfrastructureIsInSync()
result.AssertManagedCount(2)
```

Reply:
True, this is more straightforward, fixed